libs/natparse/src/main/java/org/amshove/natparse/parsing/StatementListParser.java

```diff
@@ -2491,7 +2491,10 @@ private IfSelectionNode ifSelection() throws ParseError
 
 		var opening = consumeMandatory(statement, SyntaxKind.IF);
 		consumeMandatory(statement, SyntaxKind.SELECTION);
-		consumeAnyOptionally(statement, List.of(SyntaxKind.NOT, SyntaxKind.UNIQUE, SyntaxKind.IN, SyntaxKind.FIELDS));
+		consumeOptionally(statement, SyntaxKind.NOT);
```

Comment:
```suggestion
		if (consumeOptionally(statement, SyntaxKind.NOT))
		{
		    consumeMandatory(statement, SyntaxKind.UNIQUE);
		    consumeOptionally(statement, SyntaxKind.IN);
		    consumeOptionally(statement, SyntaxKind.FIELDS);
		}
```

Doesn't this resemble the documentation better, or is the documentation faulty here?

Documentation: ` IF SELECTION [NOT UNIQUE [IN [FIELDS]]] `